apps/code-examples/src/app/code-examples/lists/paging/with-content/demo-data.service.ts

```diff
@@ -102,7 +102,7 @@ export class DemoDataService {
       totalCount: people.length,
     }).pipe(
       // Simulate network latency.
-      delay(1000),
+      delay(100),
```

Comment:
why are we reducing this?

Reply:
I was worried about this delay causing the sample tests I was writing to time out. 100ms is still a pretty good length of delay.

Reply:
looking at the code example, visually this is way too short of a change imo

Reply:
reverted